modules/@apostrophecms/schema/ui/apos/components/AposSchema.vue

```diff
@@ -4,10 +4,43 @@
 <template>
   <div class="apos-schema">
     <div
-      v-for="field in schema" :key="field.name"
+      v-for="(field, index) in schemaToParse"
+      :key="`${field.name}${index}`"
       :data-apos-field="field.name"
     >
+      <div
+        v-if="field.type === 'alignedFields'"
+        class="apos-schema__aligned-block"
+      >
+        <label class="apos-field__label">
+          {{ field.label }}
+        </label>
+        <div class="apos-schema__aligned-fields">
+          <div
+            class="apos-schema__aligned-field"
+            v-for="subField in field.fields"
+            :key="subField.name"
+            :data-apos-field="subField.name"
+          >
+            <component
```

Comment:
I'm not a huge fan of repeating these components twice but I don't see a perfect way to do it. Extracting it in another component would force us to still pass a lot of props each time.